apollo-router/src/plugins/telemetry/apollo_exporter.rs

```diff
@@ -240,17 +240,10 @@ impl ApolloExporter {
         let mut has_traces = false;
 
         for (_, traces_and_stats) in proto_report.traces_per_query.iter_mut() {
-            if !traces_and_stats.trace.is_empty()
-                || !traces_and_stats
-                    .internal_traces_contributing_to_stats
```

Comment:
Just to be clear this is as far as I can tell some unnecessarily conservative code that clears data that is never currently set.